### DISCUSSION TITLE
Share your showcases with us!

### DISCUSSION THREAD

**Discussion by chenjiahan** [web-infra-dev/modern.js#3554] [Show and tell]
Hello everyone 🙌,

We are excited to see what you have created using Modern.js! 

Please share your websites with us by leaving a comment below. Don't forget to include a link to your website or project and a brief description of what you have created.

Some examples:

- Volctrans: [https://translate.volcengine.com/](https://translate.volcengine.com/)
- Writingo: [https://writingo.net](https://writingo.net)
- Rspack: [https://rspack.dev/](https://rspack.dev/)

**Comment by chenjiahan** [comment]
Volcengine Developer - https://developer.volcengine.com/

Based on Modern.js Framework.

**Comment by chenjiahan** [comment]
- Tiktok Seller: https://seller-us-accounts.tiktok.com/
- Tiktok Steamer: https://shop.tiktok.com/streamer/welcome
- Tiktok Shop Partner: https://partner-us.tiktok.com/

**Comment by chenjiahan** [comment]
Doubao (豆包): https://www.doubao.com

**Comment by aliezzahn** [comment]
Dynamic SVG Arrows: https://dsa.hncore.website

About:  is a DOM-aware, highly flexible React component for rendering animated, curved SVG arrows between two elements or coordinates.

**Comment by lqPrototype** [comment]
ModernJs 3.0 + Module Federation 2.0: https://github.com/lqPrototype/module-federation-modern

**Comment by lvscar** [comment]
fitting.icu  :  https://fitting.icu
A bicycle fitting tool that runs entirely within browser.
